### DISCUSSION TITLE
No spending limit for Hobby users

### DISCUSSION THREAD

**Discussion by GitonioDev** [vercel/community#7358] [Help] [answered]
### Summary

Hey There, i am currently using a "Hobby" Plan on Vercel, and i ask me if i can get charged for exceeding the Limits like, Bandwith, edge functions or Functions. Because i cant see a option to enable Spending Limit

### Steps to Reproduce

-

**Comment by swarnava** [comment, marked as answer]
You can not be charged on the Hobby (free) plan, so Spend Management is not necessary. If you exceed the included usage of the free plan, your sites will be automatically paused.

[a comment by an automated account is not shown]
